BraveWallet/Crypto/Transaction Confirmations/TransactionConfirmationView.swift

```diff
@@ -0,0 +1,285 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveCore
+import BraveUI
+import struct Shared.Strings
+
+struct TransactionConfirmationView: View {
+  var transactions: [BraveWallet.TransactionInfo]
+  @ObservedObject var networkStore: NetworkStore
+  
+  @Environment(\.sizeCategory) private var sizeCategory
+  @Environment(\.presentationMode) @Binding private var presentationMode
+  
+  private enum ViewMode: Int {
+    case transaction
+    case details
+  }
+  
+  @State private var viewMode: ViewMode = .transaction
+  @State private var activeTransactionIndex: Int = 0
+  
+  private func next() {
+    activeTransactionIndex += 1
+    if activeTransactionIndex == transactions.endIndex {
+      activeTransactionIndex = 0
+    }
+  }
+  
+  private func rejectAll() {
+    presentationMode.dismiss()
+  }
+  
+  private var activeTransaction: BraveWallet.TransactionInfo {
+    transactions[activeTransactionIndex]
+  }
+  
+  var body: some View {
+    NavigationView {
+      ScrollView(.vertical) {
+        VStack {
+          // Header
+          HStack {
+            Text(networkStore.selectedChain.shortChainName)
+            Spacer()
+            if transactions.count > 1 {
+              Text("\(activeTransactionIndex + 1) of \(transactions.count)")
+                .fontWeight(.semibold)
+              Button(action: next) {
+                Text(Strings.Wallet.nextTransaction)
+                  .fontWeight(.semibold)
+                  .foregroundColor(Color(.braveBlurpleTint))
+              }
+            }
+          }
+          .font(.callout)
+          // Summary
+          VStack(spacing: 8) {
+            VStack {
+              BlockieGroup(
+                fromAddress: "0xddfabcdc4d8adc6d5beaf154f11b778f892a0740", // From Address
+                toAddress: "0xbd5b489e2177f20a0779dff0faa834ca834bcd39", // To Address
+                size: 48
+              )
+              Group {
+                if sizeCategory.isAccessibilityCategory {
+                  VStack {
+                    Text("Account 1") // From Address
+                    Image(systemName: "arrow.down")
+                    Text("0xbd5b***bcd39") // To Address
```

Comment:
These are clearly placeholder values in this file, will not comment further on needing to make the verbatim or use real values